test/test_decomp.py

```diff
@@ -422,7 +439,7 @@ def _torch_dispatch(self, func, types, args=(), kwargs=None):
                     torch.ops.aten.new_empty_strided.default,
                     torch.ops.aten.randn.default,
                     torch.ops.aten.native_dropout.default,
-                ] or any_unsupported(args, kwargs):
+                ] or any_unsupported(args, kwargs) or in_place:
```

Comment:
doesn't this change the behavior of non-recursive decomps too?

Reply:
Niet. in-place ops were not being tested, before, and those that were, happened to pass this by chance. There is no logic in this function to handle in-place functions.